AudioCompression/src/audioCompression/algorithm/FilterBankStep.java

```diff
@@ -82,6 +82,8 @@ public RawAudio reverse(Subbands input) {
 		}
 		WavAudioOutput out = 
 				new WavAudioOutput(windows, input.getWindowOverlap(), input.getSampleRate());
+		
+		out.writeFile(name);
```

Comment:
I would kind of rather we do the actual file writing either external to the pipeline (like call out.writeFile() on the object that comes out of the reverse pipeline), so that the `FilterBankStep` can remain somewhat pure, as in it does the job of a filter bank, not a filterbank AND a file writer... That OK?